README.md

```diff
@@ -3,14 +3,33 @@
 ## Objectives
 
 1. Generate URLs using route helpers
-2. Draw routes with implicit and explict names using :as
+
+2. Draw routes with implicit and explicit names using ```:as```
+
 3. Draw routes with variables
+
 4. Generate URLs using route helpers that require variables
 
-## Notes
 
-Goal of this lab is to build an index and show action using students where index links to students show and each students show links back to the index.
+## Instructions
+
+Currently four tests are currently failing and need implementation code to start passing. In order to complete this lab you will need to complete the following tasks:
+
+* Update the ```config/routes.rb``` file and utilize the ```:as``` call for the ```show``` route to create the ```student_path(:id)``` helper method.
```

Comment:
Also when you give the order of operations, I try and go from easiest thing to hardest. In the same way that when you're playing games the levels go up increasing levels of difficulty